cloud/pkg/router/provider/rest/rest.go

```diff
@@ -201,7 +201,8 @@ func (r *Rest) GoToTarget(data map[string]interface{}, stop chan struct{}) (inte
 	if !ok || len(content) == 0 {
 		return nil, errors.New("invalid convert to []byte")
 	}
-	req, err := httpUtils.BuildRequest(http.MethodPost, r.Endpoint, bytes.NewReader(content), "", "")
+	nodeName, _ := data["nodeName"].(string)
```

Comment:
Type confusion is shown here, this should be avoided by checking for an error when type casting.
```suggestion
	nodeName, ok := data["nodeName"].(string)
	if !ok {...
```

Reply:
ok